src/login/login.container.js

```diff
@@ -0,0 +1,56 @@
+import React, { PropTypes } from 'react'
+import GoogleLogin from 'react-google-login'
+import { connect } from 'react-redux'
+import * as actions from './login.actions'
+import { bindActionCreators } from 'redux'
+
+export class LoginButton extends React.Component {
+
+    render () {
+        if (this.props.user.id) {
+            return (
+                <div>
+                    {this.props.user.fullname}<br/> Crédits : {this.props.user.credit}<br/>
+                    <button onClick={this.props.handleLogout}>Logout</button>
+                </div>
+            )
+        } else {
+            return (
+                <GoogleLogin
+                    clientId="673157831962-gcgp4mj9mgadau0nh9pbaikhbmqkl04d.apps.googleusercontent.com"
```

Comment:
Shall we put this in a constant?
Using:
```javascript
config.googleKey = 
if (!process.env.BROWSER) { //webpack will tree-shake this section, allowing us to use the same config file on the client and server side
  config.secretServerPassword = 'XYZ'
}
```
Allow us to share the same config file for the client and the server :)